### ISSUE TITLE
hyper-sync-settings error

### ISSUE THREAD

**Issue by katiehartrick** [dfrankland/hyper-sync-settings#222]
Error message: Git could. not be found at the expected path: `Users/{username}/.hyper_plugins/node_modules/dugite/git/bin...`

This is a fresh install on a fresh macOS running 10.15.7. Have created personal access token + gist. 

Most plugin features work (creating a .hyper.js file inside .hyper-sync-settings folder and copying settings work). Syncing with github does not work. 

<img width="341" alt="hyper-sync-settings-error" src="https://user-images.githubusercontent.com/6202772/111927544-17bb5500-8aec-11eb-8a4c-ae603bcb1476.png">

**Comment by dfrankland** [comment]
Yeah, there's an issue with how Hyper installs `dugite`. I bet if we update that dependency the problem will resolve itself. I don't even use Hyper anymore, so I am open to someone making a PR for this.
